gradoop-data-integration/src/main/java/org/gradoop/dataintegration/transformation/functions/SetBasedLabelFilter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.dataintegration.transformation.functions;
+
+import org.gradoop.common.model.api.entities.EPGMElement;
+import org.gradoop.flink.model.impl.functions.filters.CombinableFilter;
+
+import java.util.Set;
+
+/**
+ * A simple Filter which only excepts entries that are present in a given {@link Set}.
+ *
+ * @param <T> The {@link EPGMElement} the filter is used for.
+ */
+public class SetBasedLabelFilter<T extends EPGMElement> implements CombinableFilter<T> {
```

Comment:
We thought about this two. However, since this PR is related to transformations we decided to include this in another PR which includes new constructor + use.

Your decision, how you like it better?

Reply:
Since you are using the LabelIsIn filter in other classes of your transformations, I think we should use it here in this class too, if you will agree. Adding the constructor is quiet simple because the internal representation of the labels is a already a collection of strings:
```
public LabelIsIn(Set<String> labels) {
    this.labels = labels;
  }
```

Reply:
Yes, we know that. So @p-f please add the constructor in this PR.